app/components/post_draft/send_action/scheudled_post_tooltip.tsx

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Can you correct the spelling of this file?

Reply:
Done.